src/components/LayoutHeader/Nav/LinkItems/index.tsx

```diff
@@ -109,8 +110,8 @@ const LinkItems: React.FC = () => {
             key={i}
             className={styles.linkItem}
             ref={popup?.containerEl}
-            onMouseEnter={popup?.open}
-            onMouseLeave={popup?.close}
+            onMouseEnter={isMobile ? undefined : popup?.open}
+            onMouseLeave={isMobile ? undefined : popup?.close}
```

Comment:
While I'm not incredibly enthused by having to do a JS mobile check just for this, but I see why it's needed with how `mouseEnter` and `click` trigger at the same time on touch.

I think maybe a small debounce on all popover changes would slightly better handle edge-cases such as mobile devices (or other devices detected as mobile) with mice or desktop devices with touch, but that's a minor enough case that I think we can let this through.